### PR TITLE
Connect text feature with back end

### PR DESCRIPTION
**Pull request by jetfuel**

### CODE REVIEW

**Comment by jetfuel** [comment]
Fixes https://github.com/PaddlePaddle/VisualDL/issues/246

File: visualdl/logic/sdk.h [hunk 1]
```diff
@@ -304,6 +309,9 @@ struct Text {
   Tablet tablet_;
 };
 
+/*
+ * Text reader.
```
Comment (daming-lu) [inline visualdl/logic/sdk.h:313]: Do we need this comment?

File: visualdl/server/lib.py [hunk 2]
```diff
@@ -159,6 +159,43 @@ def get_histogram_tags(storage):
...
+        ids = texts.ids()
+        timestamps = texts.timestamps()
+
+        data = list(zip(timestamps, ids, records))
+        data_size = len(data)
+
+        if data_size <= num_records:
+            return data
+
+        span = float(data_size) / (num_records - 1)
```
Comment (daming-lu) [inline visualdl/server/lib.py:180]: if num_records == 1, it would 'div by 0'

**Review by daming-lu — DISMISSED** [review]
only 2 comments 😄

File: visualdl/logic/sdk.h (hunk 1 above)
Comment (jetfuel) [reply visualdl/logic/sdk.h:313]: This is to match with the comment style in other classes.

File: visualdl/server/lib.py (hunk 2 above)
Comment (jetfuel) [reply visualdl/server/lib.py:180]: The num_records is like a constant, where is currently set to 100. 
I will move it away from the parameter and set it into a const

**Comment by jetfuel** [comment]
@daming-lu Code, updated.

**Review by daming-lu — APPROVED** [review]

**Pull request merged**